### ISSUE TITLE
Login Screen - Screen reader does not read error from bad login/password and is not keyboard accessible

### ISSUE THREAD

**Issue by jeromeleecms** [18F/cms-hitech-apd#1303]
Screen reader is not reading content / not accessible
- No error message reported when no login information is provided
- Error Message when submitting bad/no password is not reported by screen reader (also not keyboard accessible)

![image.png](https://images.zenhubusercontent.com/5c547884d11206601037f0c7/f1c566cf-85e0-4519-bac0-3c02ca940f86)

### This task is done when...
 - [ ] Screen Reader reports error when submitting bad or incomplete login/password
- [ ] The error message provides a meaningful error message other than "Sorry! Something went wrong"
- [ ] Keyboard can access error message

**Comment by jeromeleecms** [comment]
@nkkl could you please take a look to see if this in the ballpark for what is expected for testing? We can close this out and/or move the content into a general Google doc for review once the design system + table changes have been made (or we can keep this pending and remove the assignment).

**Comment by nkkl** [comment]
You are exactly right about the bug! Great catch.  
No opinion about how we track it.

**Comment by jeromeleecms** [comment]
I'm just going to drop this at the bottom of the prioritized backlog then. James is going to work on reviewing the error messaging (added dependency above) and then I can retest.

**Comment by mgwalker** [comment]
Since #1376 was merged that changed the login page styles and the way errors are displayed, we should check if this is still a problem.

**Comment by jeromeleecms** [comment]
@NAretakis @hbillings Is this one good to close? It seems #1679 was merged to help address this, but I don't have a way to verify if this is still a bug...

**Comment by hbillings** [comment]
This was fixed!
